pkgdown/_pkgdown.yml

```diff
@@ -26,11 +26,18 @@ reference:
   contents:
     - sass
     - sass_options
-    - sass_cache_options
 - title: Sass code
   description: |
     Tools for generating Sass code from R
   contents:
     - sass_file
     - sass_layer
     - as_sass
+- title: Caching
+  description: |
+    Functions related to caching sass output
+  contents:
+    - sass_file_cache
+    - sass_get_default_cache
```

Comment:
I think you need to add `sass_set_default_cache`?

Reply:
They have the same `@rdname`, and the pkgdown check seems to be happy with it the way it is.